### PR TITLE
fix the layout tests

### PR DESCRIPTION
**Pull request by jf2048**
Part of https://github.com/gtk-rs/gtk-rs-core/issues/64, only really needed to pick up `gio-unix-2.0`. Also fixes those warnings about `PACKAGES`, etc being unused on windows

### CODE REVIEW

**Review by sdroege — APPROVED** [review]

**Review by bilelmoussaoui — APPROVED** [review]

**Comment by jf2048** [comment]
BTW I think this means the tests will still just be broken on windows due to not having `gio-unix`? Related to #1052

File: src/codegen/sys/tests.rs [hunk 1]
```diff
@@ -145,7 +145,11 @@ fn prepare_cconsts(env: &Env) -> Vec<CConstant> {
                     // see the GValue machinery around them for example
                     constants.push(CConstant {
                         name: format!("(guint) {}", member.c_identifier),
-                        value: member.value.clone(),
+                        value: member
+                            .value
+                            .parse::<i32>()
```
Comment (sdroege) [inline src/codegen/sys/tests.rs:150]: Are you sure these can never be an actual `u32` that is bigger than `i32::MAX`?

Comment (jf2048) [reply src/codegen/sys/tests.rs:150]: The `unwrap_or_else` should cover that?

Comment (sdroege) [reply src/codegen/sys/tests.rs:150]: True

**Pull request merged**